mkdocs.yml

```diff
@@ -21,11 +21,16 @@ nav:
       - Glossary: 'overview/glossary.md'
       - Logo assets: 'overview/logos.md'
   - Getting Started:
-      - Getting Started: 'getting-started/index.md'
-      - Create an App: 'getting-started/create-an-app.md'
-      - App configuration:
+      - Installing a standalone server (Developer or Admin): 'getting-started/standalone-install.md'
+      - Configuring Backstage (Admin):
```

Comment:
I think that it probably makes sense to drop the audience parentheses in the titles for the docs. But keep the audience in the actual documentation in the preamble